### DISCUSSION TITLE
How to load a video source instead of webcam source?

### DISCUSSION THREAD

**Discussion by tolotratlt** [asus4/tf-lite-unity-sample#361] [Q&A]
Hi 
How can I use a video source (maybe from a video player or through a RenderTexture) instead of a webcam source?
I especially want to use the BlazePose project for this.

**Comment by tolotratlt** [comment]
I've just found it!
I need to create a Video ScriptableObject. Then, I can set the video data through the video element source of the ScriptableObject.
After that, I can use this ScriptableObject as the Source or Editor Source for the Virtual Texture Source object that will be used with Blazepose.

However, how can I control the video playback, especially seeking within the video?

**Reply by asus4** [reply]
@tolotratlt The [VideoTextureSource](https://github.com/asus4/TextureSource/blob/main/Packages/com.github.asus4.texture-source/Runtime/VideoTextureSource.cs) doesn't have any playback control APIs for now. Please implement your custom ScriptableObject inheriting the `BaseTextureSource`, or making an entirely custom system might be easier. Blaze Pose receives any texture.
